### PR TITLE
Pipelines 1.5 cli updated

### PR DESCRIPTION
**Pull request by ktania46**
This is for the upcoming Pipelines 1.5 GA

Preview link :  https://deploy-preview-33810--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html

@ppitonak  Can you please review this. 
Thanks !

cc @Preeticp @SNiemann15

### CODE REVIEW

File: modules/op-installing-tkn-on-linux.adoc [hunk 1]
```diff
@@ -12,11 +12,11 @@ For Linux distributions, you can download the CLI directly as a `tar.gz` archive
 
 . Download the relevant CLI.
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.17.2/tkn-linux-amd64-0.17.2.tar.gz[Linux (x86_64, amd64)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.19/tkn-linux-amd64-0.19.tar.gz[Linux (x86_64, amd64)]
```
Comment (VeereshAradhya) [inline modules/op-installing-tkn-on-linux.adoc:15]: Due to a regression bug fix the version of `tkn` got changed from `0.19` to `0.19.1`. Links needs to be updated

File: modules/op-installing-tkn-on-linux.adoc [hunk 2]
```diff
@@ -12,11 +12,11 @@ For Linux distributions, you can download the CLI directly as a `tar.gz` archive
...
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.17.2/tkn-linux-s390x-0.17.2.tar.gz[Linux on IBM Z and LinuxONE (s390x)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.19/tkn-linux-s390x-0.19.tar.gz[Linux on IBM Z and LinuxONE (s390x)]
```
Comment (VeereshAradhya) [inline modules/op-installing-tkn-on-linux.adoc:17]: same here

File: modules/op-installing-tkn-on-linux.adoc [hunk 3]
```diff
@@ -12,11 +12,11 @@ For Linux distributions, you can download the CLI directly as a `tar.gz` archive
...
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.17.2/tkn-linux-ppc64le-0.17.2.tar.gz[Linux on IBM Power Systems (ppc64le)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.19/tkn-linux-ppc64le-0.19.tar.gz[Linux on IBM Power Systems (ppc64le)]
```
Comment (VeereshAradhya) [inline modules/op-installing-tkn-on-linux.adoc:19]: same here

File: modules/op-installing-tkn-on-windows.adoc [hunk 4]
```diff
@@ -10,7 +10,7 @@ For Windows, the `tkn` CLI is provided as a `zip` archive.
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.17.2/tkn-windows-amd64-0.17.2.zip[CLI].
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.19/tkn-windows-amd64-0.19.zip[CLI].
```
Comment (VeereshAradhya) [inline modules/op-installing-tkn-on-windows.adoc:13]: same here

File: modules/op-installing-tkn-on-macos.adoc [hunk 5]
```diff
@@ -10,7 +10,7 @@ For macOS, the `tkn` CLI is provided as a `tar.gz` archive.
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.17.2/tkn-macos-amd64-0.17.2.tar.gz[CLI].
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.19/tkn-macos-amd64-0.19.tar.gz[CLI].
```
Comment (ktania46) [inline modules/op-installing-tkn-on-macos.adoc:13]: @VeereshAradhya  here too ?

Comment (SNiemann15) [reply modules/op-installing-tkn-on-macos.adoc]: @ktania46 I would say yes in the previous releases they were the same through all platforms.

**Review by Preeticp — APPROVED** [review]
/lgtm

**Pull request merged**